.github/actions/api/breaking-change/entrypoint.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+set -e
+
+readonly base="$1"
+readonly revision="$2"
+readonly fail_on_diff="$3"
+readonly include_checks="$4"
+echo "running oasdiff breaking base: $base, revision: $revision, fail_on_diff: $fail_on_diff, include_checks: $include_checks"
+
+go install github.com/treeverse/oasdiff@v1.8.2
```

Comment:
This can be slow.  Perhaps a Docker image?  (Doesn't need to happen in the first revision, of course.)